pkg/combustion/registry.go

```diff
@@ -173,6 +169,28 @@ func createRegistryCommand(ctx *image.Context, commandName string, args []string
 	return cmd, logFile, nil
 }
 
-func IsEmbeddedArtifactRegistryEmpty(registry image.EmbeddedArtifactRegistry) bool {
-	return len(registry.HelmCharts) == 0 && len(registry.ContainerImages) == 0
+func IsEmbeddedArtifactRegistryAndKubernetesManifestsEmpty(ctx *image.Context) bool {
```

Comment:
This is now too complex. I'd suggest to invert the naming and usage.
```suggestion
func IsEmbeddedArtifactRegistryConfigured(ctx *image.Context) bool {
```

Reply:
Will do👍